org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/FileHandler.java

```diff
@@ -192,30 +217,20 @@ protected boolean createDirectories(String dest) throws IOException {
 	protected CommandStatus executeTransfer(final String destination) {
 		// Execute the file transfer
 		transferStatus = command.execute();
-
+		
 		// Check that the move succeeded
 		boolean check = false;
 		try {
 			check = exists(destination);
 		} catch (IOException e) {
 			e.printStackTrace();
```

Comment:
If action is not going to be taken here, then the exception should be passed to the caller.

Reply:
Moved the exception checking up to the publicly called function.